test/gibct/containers/LandingPage.unit.spec.js

```diff
@@ -0,0 +1,30 @@
+import { createStore } from 'redux';
+import { expect } from 'chai';
+import React from 'react';
+import sinon from 'sinon';
+import SkinDeep from 'skin-deep';
+
+import { LandingPage } from '../../../src/js/gi/containers/LandingPage';
+import reducer from '../../../src/js/gi/reducers';
+
+const store = createStore(reducer);
```

Comment:
In the test for `GiBillApp`, you set `defaultProps` in one line instead referring to it later as `store.getState()`. Probably should be consistent here (I prefer the `defaultProps`).